Compile.ps1

```diff
@@ -97,20 +97,7 @@ Get-ChildItem .\config | Where-Object {$psitem.extension -eq ".json"} | ForEach-
 
 $xaml = (Get-Content .\xaml\inputXML.xaml).replace("'","''")
 
-# Dot-source the Get-TabXaml function
-. .\functions\private\Get-TabXaml.ps1
-
-Update-Progress "Building: Xaml " 75
-$appXamlContent = Get-TabXaml "applications" 5
-$tweaksXamlContent = Get-TabXaml "tweaks"
-$featuresXamlContent = Get-TabXaml "feature"
-
-
 Update-Progress "Adding: Xaml " 90
```

Comment:
```suggestion
```
If you're converting the code generation from static (using `Get-TabXaml`) to Dynamic (using your new `Invoke-WPFUIElements` function).. then the progress status of `Adding: Xaml` doesn't make sense when Compiling.

Reply:
left it bc of this; `$script_content.Add($(Write-output "`$inputXML =  '$xaml'"))` but agree that it does not make much sense, thanks for pointing out :)

Reply:
I didnt have this error anyways...